adafruit_sharpmemorydisplay.py

```diff
@@ -85,23 +85,26 @@ def show(self):
 
         with self.spi_device as spi:
 
+            _imageBuf = bytearray()
```

Comment:
Can you change this variable image to something like `image_buffer` to appease the linter?  Thanks!

Reply:
Done.